packages/redis-message-deduplication-store/lib/RedisConsumerMessageDeduplicationStore.ts

```diff
@@ -0,0 +1,60 @@
+import type { ConsumerMessageDeduplicationStore } from '@message-queue-toolkit/core'
+import type { Redis } from 'ioredis'
+
+export type RedisConsumerMessageDeduplicationStoreDependencies = {
+  redis: Redis
+}
+
+export type RedisConsumerMessageDeduplicationStoreConfig = {
+  keyPrefix?: string
+}
+
+export class RedisConsumerMessageDeduplicationStore implements ConsumerMessageDeduplicationStore {
+  private readonly redis: Redis
+  private readonly config: RedisConsumerMessageDeduplicationStoreConfig
+
+  constructor(
+    dependencies: RedisConsumerMessageDeduplicationStoreDependencies,
+    config: RedisConsumerMessageDeduplicationStoreConfig,
+  ) {
+    this.redis = dependencies.redis
+    this.config = config
+  }
+
+  async setIfNotExists(key: string, value: string, ttlSeconds: number): Promise<boolean> {
```

Comment:
can you add jsdoc, explaining what resulting boolean means? (it's unclear whether it indicates if value existed before, was set, or if operation errored out)